templates/index.html

```diff
@@ -18,14 +18,49 @@
 </head>
 
 <body>
+	<div id="container">
+		<div id="header">
+			<h1>Zombie Server</h1>
+			<hr id="hr1" />
+			<hr id="hr2" />
+			<hr id="hr3" /> 
+			<hr id="hr4" />
+			</br>
+		</div> <!--close header-->
+		<div id="navbar">
+			<ul id="navdesign"> <!--list of links to other pages-->
+				<li><a href="index.html" class="selected">Home</a></li> <!--class selected means you are on that page-->
+				<!--<li><a href="loadSave.html">Load & Save</a></li>
+				<li><a href="log.html">Log</a></li>-->
+			</ul>
+			<div id="navdesignbar"></div> <!--black design at bottom of navigation bar-->
+		</div> <!--close navbar-->
+		<div id="content">
+		
+		</div> <!--clsoe content-->
+		<div class="footer">
+			<p class="footer">Designed by Miten Chauhan and Callum Gill</p>
+		</div> <!--close footer-->
+	</div> <!--close container-->
 
-  <div class="template">
+  
+
+</body> 
+</html>
+
+
+
+<!--old code, just incase it is required in future-->
+
+<!--<div class="template">
         {% block content %}
-        Welcome to the test page of <a href="http://github.com/muneeb-ali/apache-flask">apache-flask</a>.<br><br>
+        Welcome to the test page of the fucking ZOMBIE ANIMATRONICS SERVER<a href="http://github.com/muneeb-ali/apache-flask">apache-flask</a>.<br><br>
```

Comment:
Ha!
Unfortunately this repo is public facing, so profanity is perhaps out of place.